docs/FlowCollector.md

```diff
@@ -4007,6 +4007,15 @@ ebpf describes the settings related to the eBPF-based flow reporter when the "ag
           Debug allows setting some aspects of the internal configuration of the eBPF agent. This section is aimed exclusively for debugging and fine-grained performance optimizations (for example GOGC, GOMAXPROCS env vars). Users setting its values do it at their own risk.<br/>
         </td>
         <td>false</td>
+      </tr><tr>
+        <td><b>enableTCPDrop</b></td>
+        <td>boolean</td>
+        <td>
+          enable TCP drop flows logging feature<br/>
+          <br/>
+            <i>Default</i>: false<br/>
```

Comment:
Any reason why this is not enabled by default? The overhead should not be huge, and this is a very useful feature for customers. If somehow doesn't want that he can just disable.

Reply:
we can for sure make the default on just felt it might be less overhead/risk to start with it as off by default and based on feedback we can decide

Reply:
I didn't realize it requires privileged / mounting kernel debug FS. That could be a reason to not turn it on by default :-(

Reply:
yes